Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocList.css

```diff
@@ -1,594 +1,354 @@
-  color: Black;
-  border: solid 1px #569CE0;
-  background-color: #C5E0FC;
+  margin-right: var(--remotion-themed-padding);
 }
 
-/* Workaround for IE-issue with a span inside an anchor tag blocking the :active pseudo class. */
-body.msie tr.bocListDataRow td.bocListDataCell > a
+div.bocList div.bocListTableBlock div.bocListNavigator > span > input
 {
-  position: relative;
+  min-height: unset;
+  height: 16px;
+  padding: calc(0.5 * var(--remotion-themed-padding));
+  margin: calc(0.5 * var(--remotion-themed-padding)) var(--remotion-themed-padding);
+  text-align: right;
 }
 
-/* Workaround for IE-issue with a span inside an anchor tag blocking the :active pseudo class. */
-body.msie tr.bocListDataRow td.bocListDataCell > a::before
+div.bocList div.bocListTableBlock div.bocListNavigator img
 {
-  content: "";
-  position: absolute;
-  left: 0;
-  right: 0;
-  top: 0;
-  bottom: 0;
+  height: 16px;
+  width: 16px;
+  border: var(--remotion-themed-border-width) solid transparent;
+  border-radius: 2px;
+  padding: calc(0.5 * var(--remotion-themed-padding));
+  margin: calc(0.5 * var(--remotion-themed-padding));
+  background-color: white;
 }
 
-/*  Style applied to the disabled anchor tags during hovering in BocList's data rows.  */
-tr.bocListDataRow td.bocListDataCell a.disabled:hover,
-tr.bocListDataRow td.bocListDataCell a.disabled:active
+div.bocList div.bocListTableBlock div.bocListNavigator a[href] img
 {
-  color: Gray;
-  border: 1px solid transparent;
+  border-color: var(--color-standard-border);
 }
 
-/* Style applied to SPAN elements in command-enabled column cells */
-tr.bocListDataRow td.bocListDataCell span.CommandText
+div.bocList div.bocListTableBlock div.bocListNavigator a[href]:hover img
 {
-  vertical-align: middle;
+  border-color: var(--color-primary);
+  background-color: var(--color-highlight);
 }
 
-
-/* Style to override the BocList's anchor style for drop down menus rendered inside a cell. */
-td.bocListDataCell div ul.DropDownMenuOptions li a
+div.bocList div.bocListTableBlock div.bocListNavigator a:not([href]) img
 {
-  display: block;
+  filter: opacity(0.5);
 }
+/*endregion*/
 
-td.bocListDataCell div ul.DropDownMenuOptions li.DropDownMenuItemDisabled.selected a,
-td.bocListDataCell div ul.DropDownMenuOptions li.DropDownMenuItemDisabled a:active,
-td.bocListDataCell div ul.DropDownMenuOptions li.DropDownMenuItemDisabled a:focus,
-td.bocListDataCell div ul.DropDownMenuOptions li.DropDownMenuItemDisabled a:hover
+/*region bocListMenuBlock*/
+div.bocList div.bocListMenuBlock
 {
-  background-color: #ECEEEC;
-  border-style: none;
-  padding: 2px;
-  color: gray;
+  grid-column: 2;
+  grid-row: 1;
+  background-color: var(--boclist-listmenu-background-color);
 }
 
-td.bocListDataCell div span.DropDownMenuSelect a.DropDownMenuLabel,
-td.bocListDataCell div span.DropDownMenuSelect:focus a.DropDownMenuLabel,
-td.bocListDataCell div span.DropDownMenuSelect:hover a.DropDownMenuLabel
+div.bocList div.bocListMenuBlock > div
```

Comment:
please check the renders/markup and move the hardcoded paddings/marins from c# into the  css.